src/graphql/resolvers/monsterResolver.ts

```diff
@@ -33,21 +47,21 @@ const resolveDamage = async damage => {
 };
 
 const Monster = {
-  condition_immunities: async monster =>
+  condition_immunities: async (monster: Monster) =>
     await ConditionModel.find({
       index: { $in: monster.condition_immunities.map(ci => ci.index) },
     }).lean(),
-  forms: async monster =>
+  forms: async (monster: Monster) =>
     monster.forms
       ? await MonsterModel.find({ index: { $in: monster.forms.map(f => f.index) } }).lean()
       : null,
-  legendary_actions: async monster => {
+  legendary_actions: async (monster: Monster) => {
     const { legendary_actions } = monster;
     if (!legendary_actions) return null;
 
     const resolvedLegendaryActions = [];
     for (const legendaryAction of legendary_actions) {
-      const resolvedLegendaryAction = { ...legendaryAction };
+      const resolvedLegendaryAction: Record<any, any> = { ...legendaryAction };
```

Comment:
`any` shouldn't be the record key.

Reply:
I think I got tired of writing a bunch of types. I'll fix this.